cluster/environments/azure-simple/terraform.tfvars

```diff
@@ -5,6 +5,11 @@ resource_group_location="westus2"
 cluster_name="cluster-name"
 agent_vm_count = "3"
 dns_prefix="dns-prefix"
+address_space="172.20.0.0/16"
+subnet_prefixes=["172.20.0.0/20"]
+service_CIDR="172.21.0.0/16"
+dns_IP="172.21.0.10"
```

Comment:
These values do not match up to the default vars below.  In general, would prefer to keep the `azure-simple` have as little possible needed configurable values as possible, so perhaps just set the defaults to the same as the defaults from the aks module in the `variables.tf` file and remove what's here in the `terraform.tfvars` file.